src/types/actions/block-action.ts

```diff
@@ -237,6 +237,14 @@ export interface BlockAction<ElementAction extends BasicElementAction = BlockEle
 
   // this appears in the block_suggestions schema, but we're not sure when its present or what its type would be
   app_unfurl?: any;
+
+  // exists for enterprise installs
+  is_enterprise_install?: boolean;
```

Comment:
I confirmed that `button` gets `is_enterprise_install` & `enterprise: {id: '', name: ''}`. Not sure if we should confirm the payloads for the other block elements in an enterprise (&| org install) world.